runtime/vm/visible.c

```diff
@@ -343,6 +355,17 @@ loadAndVerifyNestHost(J9VMThread *vmThread, J9Class *clazz, UDATA options)
 		/* If a problem occurred in nest host verification then the nest host value is invalid */
 		if (J9_VISIBILITY_ALLOWED == result) {
 			clazz->nestHost = nestHost;
+		} else if (J2SE_VERSION(vmThread->javaVM) >= J2SE_V15) {
+			/** 
+			 * JVM spec updated in Java 15:
+			 * M has a NestHost attribute, and its host_class_index item is used as an index into the run-time constant pool of M. The symbolic reference at that 
+			 * index is resolved (�5.4.3.1). If resolution of the symbolic reference fails, then M is its own nest host. Any exception thrown as a result of failure 
+			 * of class or interface resolution is not rethrown.
```

Comment:
Can you paraphrase this rather than directly including the JVMS quote here?

Reply:
Fixed.